language/ir-testsuite/tests/signer/keyword.mvir

```diff
@@ -0,0 +1,7 @@
+module M {
+    foo(signer: address) {
+        return;
+    }
+}
+
+// check: ParserError: Invalid Token
```

Comment:
Couple more tests for resource-y ness of signer that might be useful:

```
copy_signer_bad1(s:  signer): (signer * signer) {
  return (copy(s), move(s))
}

copy_signer_bad2(v: vector<signer>): vector<signer> {
  let s: signer;
  s = *Vector::borrow_loc(&v, 0);
}
```

Reply:
Ah, yes forgot bout copies. I'll add these